Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -615,6 +630,11 @@ protected class ClassWriter : IClassWriter {
 
     protected override string TypeInitializationValue(Type type, ConcreteSyntaxTree wr, IToken tok,
         bool usePlaceboValue, bool constructTypeParameterDefaultsFromTypeDescriptors) {
+
+      if (usePlaceboValue) {
+        return "None";
+      }
+
```

Comment:
This should be redundant due to L640. I wonder what's going on, if it's necessary.